src/trivyHelper.ts

```diff
@@ -128,3 +129,44 @@ function getTrivyDownloadUrl(trivyVersion: string): string {
             throw new Error(util.format("Container scanning is not supported on %s currently", curOS));
     }
 }
+
+export function printFormattedOutput() {
+    const trivyOutputJson = getTrivyOutput();
+    let rows = [];
+    let titles = ["VULNERABILITY ID", "PACKAGE NAME", "SEVERITY", "DESCRIPTION"];
+    rows.push(titles);
+    trivyOutputJson.forEach(ele => {
+        if (ele && ele["Vulnerabilities"]) {
+            ele["Vulnerabilities"].forEach((cve: any) => {
+                let row = [];
+                row.push(cve["VulnerabilityID"]);
+                row.push(cve["PkgName"]);
+                row.push(cve["Severity"]);
+                row.push(cve["Description"]);
+                rows.push(row);
+            });
+        }
+    });
+    
+    let config = {
+        columns: {
+          0: {
+            width: 25,
+            wrapWord: true
+          },
+          1: {
+            width: 25,
+            wrapWord: true
+          },
+          2: {
+            width: 25,
+            wrapWord: true
+          },
+          3: {
+            width: 65,
+            wrapWord: true
+          }
+        }
+      };
+    console.log(table.table(rows, config));
```

Comment:
This code is duplicate. Move it to utils

Reply:
Moved